JS-Classes/Navigation.js

```diff
@@ -0,0 +1,52 @@
+export default class Navigation {
+  constructor() {
+    this.navigation = document.querySelector('nav');
+  }
+
+  /* eslint-disable class-methods-use-this */
```

Comment:
- [ OPTIONAL ] I will suggest that you avoid disabling linters for errors such as these because using the `eslint-disable` directive to ignore the `class-methods-use-this` rule can lead to potential issues in the code. It's always better to understand why the rule is being triggered and address the underlying problem rather than suppressing the warning altogether. :100: